charts/rawls/templates/migration/job.yaml

```diff
@@ -0,0 +1,88 @@
+metadata:
+  name: rawls-liquibase-migration
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocr.argoproj.io/hook-delete-policy: BeforeHookCreation
+  labels: 
+    {{- include "rawls.labels" . | nindent 4 }}
+spec:
+  backoffLimit: 2
+  template:
+    metadata:
+      name: rawls-liquibase-migration
+      labels:
+        {{- include "rawls.labels" . | nindent 8 }}
+    spec:
+      # Allow migration-liquibase to kill migration-sqlproxy's process
+      shareProcessNamespace: true
+      restartPolicy: Never
+      serviceAccountName: rawls-migration-sa
+      hostAliases:
+      - ip: 127.0.0.1
+        hostnames:
+        - sqlproxy
+      volumes:
+      - name: app-ctmpls
+        secret:
+          secretName: {{ .Values.migration.secretPrefix }}-app-ctmpls
+      - name: sqlproxy-ctmpls
+        secret:
+          secretName: {{ .Values.migration.secretPrefix }}-sqlproxy-ctmpls
+      containers:
+      - name: migration-liquibase
+        image: "gcr.io/broad-dsp-gcr-public/rawls:{{ $imageTag }}"
+        command: ['bash', '-c']
+        # Sleep for 15s to allow CloudSQL proxy time to start up. See DDO-1284 / BT-296
+        # The `find /rawls -name 'rawls*.jar'` is from Rawls's own Dockerfile CMD
+        # References templated liquibase.properties, see https://docs.google.com/document/d/19ethQWyH29H-jUWwgFoCxKfjmzcG-NCmSgXNAUJAYaU/edit#
+        args:
+        - |-
+          sleep 15s && \
+          java -cp $(find /rawls -name 'rawls*.jar') liquibase.integration.commandline.Main \
+            --defaultsFile='/etc/liquibase.properties' \
+            --classpath="$(find /rawls -name 'rawls*.jar')" \
+            --username="$DB_USERNAME" \
+            --password="$DB_PASSWORD" \
+            {{ .Values.migration.dryRun | ternary "updateSQL" "update" }}; \
+          EXIT={{ .Values.migration.failBasedOnLiquibase | ternary "$?" "0" }}; \
+          pkill -SIGTERM cloud_sql_proxy; \
+          exit $EXIT
+        env:
+        - name: DB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: rawls-sql-secrets
+              key: db-username
+        - name: DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: rawls-sql-secrets
+              key: db-password
+        volumeMounts:
+        - mountPath: /etc/liquibase.properties
+          subPath: liquibase.properties
+          name: app-ctmpls
+          readOnly: true
+      - name: migration-sqlproxy
+        # alpine provides `sh`
+        image: gcr.io/cloudsql-docker/gce-proxy:1.23.0-alpine
+        envFrom:
+        - secretRef:
+            name: {{ .Values.migration.secretPrefix }}-sqlproxy-env
+        volumeMounts:
+        - mountPath: /etc/sqlproxy-service-account.json
+          subPath: sqlproxy-service-account.json
+          name: sqlproxy-ctmpls
+          readOnly: true
+        command: ['sh', '-c']
+        args:
+        - |-
+          /cloud_sql_proxy ${CLOUDSQL_LOGGING:-"-verbose"} \
```

Comment:
do we need verbose logging? Or is that just how it's set on our other proxies?

Reply:
That's how it is set on all other proxies. I basically reduced the entire 160 line script down to that shell command, minus the support for not passing any credentials (we'd just omit the credentials line) and the "you didn't pass a required variable" checks.